crates/chain/src/indexed_tx_graph.rs

```diff
@@ -1,6 +1,8 @@
-//! Contains the [`IndexedTxGraph`] structure and associated types.
+//! Contains the [`Indexer`] trait, the [`IndexedTxGraph`] structure,
+//! and the associated [`ChangeSet`].
```

Comment:
Dunno how valuable this module level documentation is here. Docs.rs will display a list of these just below.

Could just have instead "See [`IndexedTxGraph`] for documentation about how the components of this module relate to each other".